CONTRIBUTING.md

```diff
@@ -62,6 +62,21 @@ There are some tools that will be helpful to you in developing locally. While th
 
 First make sure you have working [golang development environment](https://learn.gopherguides.com/courses/preparing-your-environment-for-go-development) setup. Also make sure you have the latest version of `terraform` [installed.](https://learn.hashicorp.com/tutorials/terraform/install-cli)
 
+## Using a local version of opslevel-go
+
+In `go.mod` uncomment (remove the //) the line at the bottom of the file:
+
+```
+// Uncomment for local development
+//replace github.com/opslevel/opslevel-go => ./submodules/opslevel-go/
+```
+
+Now you may make changes in your local git submodule of `opslevel-go`.
```

Comment:
This worked for me at one point but stopped working. Not sure why? Am I missing something from the instructions?

Reply:
@maxim-opslevel oh ya it now needs to be
```
replace github.com/opslevel/opslevel-go/v2022 => ./submodules/opslevel-go/
```
the package name changed to appease the golang version gods